service/src/test/java/bio/terra/landingzone/library/landingzones/AzureIntegrationUtils.java

```diff
@@ -0,0 +1,96 @@
+package bio.terra.landingzone.library.landingzones;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.identity.ClientSecretCredential;
+import com.azure.identity.ClientSecretCredentialBuilder;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.assertj.core.util.Strings;
+
+public class AzureIntegrationUtils {
+  /** Path to Azure properties file. */
+  private static final String AZURE_PROPERTIES_PATH = "integration_azure_env.properties";
+
+  /** Property prefix for properties in {@link #AZURE_PROPERTIES_PATH}. */
+  private static final String AZURE_PROPERTY_PREFIX = "integration.azure";
+
+  private static final String CLIENT_ID_ENV_VAR = "AZURE_PUBLISHER_CLIENT_ID";
+  private static final String CLIENT_SECRET_ENV_VAR = "AZURE_PUBLISHER_CLIENT_SECRET";
+  private static final String TENANT_ID_ENV_VAR = "AZURE_PUBLISHER_TENANT_ID";
+
+  // 8201558-terra-dev
+  public static final AzureProfile TERRA_DEV_AZURE_PROFILE =
+      new AzureProfile(
```

Comment:
I see that is CRL pattern, but I hope this gets addressed at some point and will be moved to deployment configuration.

Reply:
It is a test config though.